include/freerdp/utils/pod_arrays.h

```diff
@@ -0,0 +1,146 @@
+/**
```

Comment:
Shouldn´t this be part of `WinPR` just like all the other array/map/... types?

Reply:
Yes could be, but as WinPR tends to mimic existing windows API...